ovn_k8s/common/config.py

```diff
@@ -0,0 +1,51 @@
+import sys
```

Comment:
Can you add a license header to all new files?  We'll need a COPYING file, as well.